backend/src/common/models/query/ITransportedQuery.ts

```diff
@@ -0,0 +1,12 @@
+import { ITrigger } from './Trigger';
+
+export interface ITransportedQuery {
+    id: string;
+    name: string;
+    code:string;
+    appId:string;
+    resourceId?:string;
+    runAutomatically?:boolean;
+    showConfirm?:boolean;
+    triggers?: Array<ITrigger>|[]
```

Comment:
fix it here and in other places 
```suggestion
    triggers?: Array<ITrigger>
```